client/components/status/status.js

```diff
@@ -239,10 +239,16 @@ export default class Status extends React.Component {
         "link" in headers && headers.link.includes("next");
       this.setState(options);
     } catch (error) {
-      logout(cookies, orgSlug);
-      toast.error(t`ERR_OCCUR`, {
-        onOpen: () => toast.dismiss(mainToastId),
-      });
+      // logout only if unauthorized or forbidden
+      if (
+        error.response &&
+        (error.response.status === 401 || error.response.status === 403)
+      ) {
+        logout(cookies, orgSlug);
+        toast.error(t`ERR_OCCUR`, {
+          onOpen: () => toast.dismiss(mainToastId),
+        });
+      }
```

Comment:
This should work, but I think this bug deserves a test, because it's very annoying and we should ensure it doesn't happen again by accident if someone in the future changes these lines and we don't catch it.